src/bson/bson-config.h

```diff
@@ -157,4 +169,22 @@
 # undef BSON_HAVE_DECIMAL128
 #endif
 
+#ifdef MS_WINDOWS
+# define BSON_HAVE_RAND_R 0
+#else
+# define BSON_HAVE_RAND_R 1
+#endif
+#if BSON_HAVE_RAND_R != 1
+# undef BSON_HAVE_RAND_R
+#endif
+
+#ifdef MS_WINDOWS
+# define BSON_HAVE_STRLCPY 0
+#else
+# define BSON_HAVE_STRLCPY 1
+#endif
```

Comment:
Let's define BSON_HAVE_STRLCPY to always be 0. manylinux1 (and travis) doesn't have it:
```
ERROR: test_bsonjs (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_bsonjs
Traceback (most recent call last):
  File "/opt/python/3.6.7/lib/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/home/travis/build/mongodb-labs/python-bsonjs/test/test_bsonjs.py", line 40, in <module>
    import bsonjs
ImportError: /home/travis/build/mongodb-labs/python-bsonjs/bsonjs.cpython-36m-x86_64-linux-gnu.so: undefined symbol: strlcpy
```

Reply:
Done.